pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -110,6 +121,7 @@ enum ProtocolVersion {
 	v8 = 8;  // Added CommandConsumerStats - Client fetches broker side consumer stats
 	v9 = 9;  // Added end of topic notification
 	v10 = 10;// Added proxy to broker
+	v11 = 11;// Added encryption_key metadata
```

Comment:
There's no need to bump up the protocol version if only adding new fields. Unknown fields are anyway ignored.

Reply:
Fixed.